test_shop.py

```diff
@@ -1,4 +1,5 @@
-# Unit testing
+# Unit testing 
```

Comment:
good naming conventions anf formatting. Tests don't pass, have another look through the ones with output to try to get them to fail (it might be with the multiple print statements)

9/14